src/DB_Command.php

```diff
@@ -456,6 +456,116 @@ public function tables( $args, $assoc_args ) {
 		}
 	}
 
+	/**
+	 * Display the database and table sizes.
+	 *
+	 * Displays the database and table sizes for `DB_NAME` specified in wp-config.php.
+	 *
+	 * ## OPTIONS
+	 *
+	 * [--db-only]
+	 * : Only display the database size without including the tables.
```

Comment:
For syntax, what do you think about this?

```
$ wp db size
# Prints database name and size as ASCII table
$ wp db size --tables
# Prints each table name and size as ASCII table
```

This would differ a little bit from what you have in that the default would be only the database size, and then you'd use the `--tables` flag to list all tables.